Dockerfile-debian.template

```diff
@@ -119,6 +119,16 @@ RUN set -ex; \
     apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
     rm -rf /var/lib/apt/lists/*
 
+# Install netcat to support healthcheck
+RUN set -ex; \
+    \
+    savedAptMark="$(apt-mark showmanual)"; \
+    \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        netcat \
```

Comment:
I would think we can/should install all of the packages at once, around line 9.

In fact, now I'm not sure why we install gnupg and dirmngr later, at line 91. @J0WI, I believe you were involved in originally writing this code in 78e3cb7a; is there a need or benefit to splitting the package installations instead of putting them all together near line 9?

Reply:
Hi @ibennetch, I created separated install because around line 9 is installed dependencies for build and install PHP modules, but I agree with you if not a problem to put in same installation

Reply:
@joubertredrat I understand, in the context of what we have now I agree that what you did makes the most sense.

Reply:
@J0WI Thank  for the informative response and recommendations!